.github/workflows/testPrerelease.yml

```diff
@@ -0,0 +1,21 @@
+name: On Tag
+on:
+  push:
+    tags:
+      - '*'
+jobs:
+  prerelease:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          path: action
+      - name: Fetch Latest Release
+        uses: thebritican/fetch-latest-release@v2.0.0
+        id: latestRelease
+      - name: Test farmersdog/release action
+        uses: './action'
+        with:
+          ghToken: ${{ secrets.GITHUB_TOKEN }}
+          chStoryUrl: ${{ secrets.CH_STORY_URL }}
```

Comment:
To confirm, this the base URL? like `https://app.clubhouse.io/farmersdog/story/`. Doesn't really matter, but I don't think that needs to be a secret.

Reply:
🤷‍♂️  Thought it'd be best for if this ever became a public repo. I don't think it's a secret either, but just in case.